### PR TITLE
KA-225 Features/task 0

### PR DESCRIPTION
**Pull request by kubmir**

### CODE REVIEW

File: NotesBackEnd/tests/Notes.Api.Tests/Notes.Api.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\packages\MSTest.TestAdapter.1.1.17\build\net45\MSTest.TestAdapter.props" Condition="Exists('..\packages\MSTest.TestAdapter.1.1.17\build\net45\MSTest.TestAdapter.props')" />
```
Comment (vit-svoboda) [inline NotesBackEnd/tests/Notes.Api.Tests/Notes.Api.Tests.csproj]: As you won't be using MSTest in your tests (I hope), you won't need this MSTest support. You can gut it from the project file and drop the 2 NuGet package references.

File: NotesBackEnd/src/Notes.Api/Notes.Api.csproj [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      <DependentUpon>Web.config</DependentUpon>
+    </None>
+    <None Include="Web.Release.config">
+      <DependentUpon>Web.config</DependentUpon>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="App_Data\" />
+    <Folder Include="Controllers\" />
+    <Folder Include="Models\" />
```
Comment (vit-svoboda) [inline NotesBackEnd/src/Notes.Api/Notes.Api.csproj]: You can delete the folders you won't be using. I believe you'll add a controller very soon, but I don't think you'll ever have models in the API project.

**Review by vit-svoboda — CHANGES_REQUESTED** [review]
Nicely done, the projects seem alright. There's just some stuff from the templates that you won't need. Also, since you went the extra mile to setup CI for your repository, it would be really cool to add a badge to your readme. Most of the CI tools offer badge generation, so it should be very simple. Let me know if you encountered an issue.

File: NotesBackEnd/tests/Notes.Api.Tests/packages.config [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSTest.TestAdapter" version="1.1.17" targetFramework="net471" />
+  <package id="MSTest.TestFramework" version="1.1.17" targetFramework="net471" />
```
Comment (vit-svoboda) [inline NotesBackEnd/tests/Notes.Api.Tests/packages.config]: With the NuGet packages, I meant these two.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,5 @@
+[![Build status](https://ci.appveyor.com/api/projects/status/apy6236i4ytwcldh?svg=true)](https://ci.appveyor.com/project/kubmir/kentico-onboarding-cs)
+[![Build status](https://ci.appveyor.com/api/projects/status/apy6236i4ytwcldh/branch/develop?svg=true&passingText=develop%20-%20OK)](https://ci.appveyor.com/project/kubmir/kentico-onboarding-cs/branch/develop)
```
Comment (vit-svoboda) [inline README.md:2]: These badges are really pretty, but you have 2 of them next to each other without any distinction what seems a bit confusing to me. On the other hand, never mind me: https://github.com/Microsoft/vscode

**Review by vit-svoboda — CHANGES_REQUESTED** [review]
Nicely done. There is just the packages.config to clear.

File: README.md (hunk 4 above)
Comment (kubmir) [reply README.md:2]: If build of develop branch will pass there will be a message in the green part of the second badge - develop - OK. The left one will be same as now.

File: NotesBackEnd/tests/Notes.Api.Tests/packages.config (hunk 3 above)
Comment (kubmir) [reply NotesBackEnd/tests/Notes.Api.Tests/packages.config]: Now, I know. I installed NUnit and found this problem. I will fix it.

**Review by vit-svoboda — APPROVED** [review]
Awesome!

**Pull request merged**